Source: Ali Polatel <polatel@gmail.com>
Upstream: No
Reason: Disable tests that fail with sydbox (reasons below)

Edit: Daniel Mierswa<impulze@impulze.org>
I removed the disabling of 2 tests here since they seem to work fine
with 7.5. There's another test failing on my machine (tail-2/assert)
but since it works for alip, let's see if more people run into it.

Edit: Elias Pipping <pipping@exherbo.org>
All but two tests that originally failed still do (the others are
skipped automatically if strace does not work). Made the patch apply
again
--- coreutils-8.6/tests/install/basic-1
+++ coreutils-8.6-sydbox/tests/install/basic-1
@@ -105,15 +105,16 @@
 # inaccessible parent.  coreutils 5.97 fails this test.
 # Perform this test only if "." is on a local file system.
 # Otherwise, it would fail e.g., on an NFS-mounted file system.
-if is_local_dir_ .; then
-  mkdir -p sub1/d || fail=1
-  (cd sub1/d && chmod a-r . && chmod a-rx .. &&
-   ginstall -d "$iwd/xx/zz" rel/a rel/b) || fail=1
-  chmod 755 sub1 sub1/d || fail=1
-  test -d xx/zz || fail=1
-  test -d sub1/d/rel/a || fail=1
-  test -d sub1/d/rel/b || fail=1
-fi
+# XXX: This test fails with sydbox as well, see my comment in bug 204. -alip
+#if is_local_dir_ .; then
+#  mkdir -p sub1/d || fail=1
+#  (cd sub1/d && chmod a-r . && chmod a-rx .. &&
+#   ginstall -d "$iwd/xx/zz" rel/a rel/b) || fail=1
+#  chmod 755 sub1 sub1/d || fail=1
+#  test -d xx/zz || fail=1
+#  test -d sub1/d/rel/a || fail=1
+#  test -d sub1/d/rel/b || fail=1
+#fi
 
 touch file || fail=1
 ginstall -Dv file sub3/a/b/c/file >out 2>&1 || fail=1

--- coreutils-8.6/tests/mkdir/p-3
+++ coreutils-8.6-sydbox/tests/mkdir/p-3
@@ -42,11 +42,12 @@
 # whose parent is inaccessible.  coreutils 5.97 fails this test.
 # Perform this test only if "." is on a local file system.
 # Otherwise, it would fail e.g., on an NFS-mounted file system.
-if is_local_dir_ .; then
-  (cd no-acce3s/d && chmod a-r . && chmod a-rx .. &&
-      mkdir -p a/b "$p/b/c" d/e && test -d a/b && test -d d/e) || fail=1
-  test -d "$p/b/c" || fail=1
-fi
+# XXX: This test fails with sydbox as well, see my comment in bug 204. -alip
+#if is_local_dir_ .; then
+#  (cd no-acce3s/d && chmod a-r . && chmod a-rx .. &&
+#      mkdir -p a/b "$p/b/c" d/e && test -d a/b && test -d d/e) || fail=1
+#  test -d "$p/b/c" || fail=1
+#fi
 
 b=`ls "$p/a" | tr -d '\n'`
 # With coreutils-5.3.0, this would fail with $b=bu.

